glowfic-dl.py

```diff
@@ -1,17 +1,23 @@
-from bs4 import BeautifulSoup
-from ebooklib import epub
-from tqdm.asyncio import tqdm
-import sys
-from urllib.parse import urljoin, urlparse
-from collections import OrderedDict
+#!/usr/bin/env python
```

Comment:
Hmm, this doesn't work for me without this change:
```suggestion
#!/usr/bin/env python3
```